src/System.Net.HttpListener/tests/HttpRequestStreamTests.cs

```diff
@@ -313,7 +313,6 @@ public void Dispose()
         }
 
         [Fact]
-        [ActiveIssue(18128, platforms: TestPlatforms.AnyUnix)] // NotSupportedException thrown instead of InvalidOperationException
```

Comment:
How did overriding FlushAsync fix this? The override is fine to add, but without it, it should have just used the base implementation, which would have queued a call to Flush.